webpack.config.js

```diff
@@ -0,0 +1,47 @@
+const entryPointNames = [
```

Comment:
Seems like this would become a pain to maintain. Can we use [`fs.readDirSync( './packages' )`](https://nodejs.org/api/fs.html#fs_fs_readdirsync_path_options) instead?